app.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+(function() {
+  angular
+  .module("toDo", [])
+  .controller("toDoController", toDoController);
```

Comment:
I would suggest calling the controller and your controller function different things, just so you (or another developer) isn't confused as to what each thing is.